server/fleet/microsoft_mdm.go

```diff
@@ -1164,9 +1221,92 @@ func (msg *SyncML) SetID(cmdID int) {
 
 // IsValid checks for required fields in the SyncML command
 func (cmd *SyncMLCmd) IsValid() bool {
-	if ((cmd.Items == nil) || (len(*cmd.Items) == 0)) && cmd.Data == nil {
+	if ((cmd.Items == nil) || (len(cmd.Items) == 0)) && cmd.Data == nil {
 		return false
 	}
 
 	return true
 }
+
+// IsEmpty checks if there are not items in the command
+func (cmd *SyncMLCmd) IsEmpty() bool {
+	if (cmd.Items == nil) || (len(cmd.Items) == 0) {
```

Comment:
Nit, you can do just `len(cmd.Items) == 0` as it's ok to call `len` on a `nil` slice (it will return 0).

Reply:
thanks, going to fix this